### DISCUSSION TITLE
Adding larger breakpoints

### DISCUSSION THREAD

**Discussion by TigMart** [tailwindlabs/tailwindcss#13724] [Help] [answered]
I want to add new screen size, but it doesn't work
no matter  that I followed  all the mentioned [instructions](https://tailwindcss.com/docs/screens#max-width-breakpoints) it doesn't work. 
 
 
```javascript
 /** @type {import('tailwindcss').Config} */
module.exports = {
  theme: {
    extend: {
      screens: {
        '3xl': '1600px',
      },
    },
  },
  plugins: [],
}
``` 
    

```html
 <div className="container">
        {/* ... */}
 </div>
```

**Comment by crswll** [comment, marked as answer]
This works. You need to provide a reproduction.

https://play.tailwindcss.com/rPtOttK8IF?layout=horizontal&file=config

**Reply by TigMart** [reply]
Thanks,  it started to work, i had an issue with syntax
